core/block_svg.ts

```diff
@@ -154,6 +155,8 @@ export class BlockSvg
    */
   relativeCoords = new Coordinate(0, 0);
 
+  private dragStrategy: IDragStrategy = new BlockDragStrategy(this);
```

Comment:
Does this mean that there is one block drag strategy instantiated per block on the page?

Reply:
Yesh, but it shouldn't have much more memory than implementing them as methods on the block itself. If you're concerned about it I can do some profiling. Theoretically we could at Yet Another Registry to deal with this, but it would be pretty messy and confusing from the external dev perspective.

Reply:
Makes sense. I agree with not adding another registry.